backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.permissions import IsAuthenticated
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class UpdateAttendeeTicketError:
+    message: str = ""
+
+
+@strawberry.type
+class TicketReassigned:
+    id: strawberry.ID
+    message: str = "Ticket was Successfully reassigned to {email}"
+
+
+UpdateAttendeeTicketResult = strawberry.union(
+    "UpdateAttendeeTicketResult",
+    (TicketReassigned, AttendeeTicket, UpdateAttendeeTicketError),
+)
+
+
+def is_ticket_owner(conference: Conference, email: str, id: str) -> bool:
+    ticket = pretix.get_user_ticket(conference, email, id)
+    return ticket is not None
+
+
+@strawberry.type
+class AttendeeTicketMutation:
+    @strawberry.mutation(permission_classes=[IsAuthenticated])
+    def update_attendee_ticket(
+        self, info: Info, code: str, input: UpdateAttendeeTicketInput
+    ) -> UpdateAttendeeTicketResult:
+        conference = Conference.objects.get(code=code)
+        try:
+            email = info.context.request.user.email
+            if not is_ticket_owner(conference, email, input.id):
+                return UpdateAttendeeTicketError(
+                    message="You are not allowed to updtae this ticket."
```

Comment:
```suggestion
                    message="You are not allowed to update this ticket."
```

Reply:
we could consider the idea of just saying "This ticket does not exist" for security purposes, but it is not super important